xml/rmt_install.xml

```diff
@@ -141,6 +141,716 @@
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Nginx</term>
+     <listitem>
+      <para>
+       A Web server configured for &rmt; routes. Having a properly configured
+       Web server allows you to target your Ingress traffic (for &rmt;) to this
+       Nginx service directly. You do not need to configure Ingress for &rmt;
+       specific paths handling, as Nginx is configured to do it.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+
+  <sect2 xml:id="sec-rmt-deploy-kubernetes-values">
+   <title>The <filename>values.yaml</filename> file</title>
+   <para>
+    The &rmt; chart is provided with the <filename>values.yaml</filename> file
+    where all parameters are documented and their default values defined. You
+    can override these values by providing your own values file, for example:
+   </para>
+<screen>&prompt.user;helm install rmtsle rmt/rmt-helm-0.1 -f myvalues-sle.yaml</screen>
+   <variablelist>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.cronjob.mirror.schedule</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"30 2 * *
+       *"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>: schedule for running the
+       <command>rmt-cli mirror</command> command (actual package mirroring)
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.cronjob.sync.schedule</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"* 1 * *
+       *"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>: schedule for running the
+       <command>rmt-cli sync</command> command (syncing with the &suse;
+       database)
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.image.pullPolicy</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"Always"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>:
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.image.repository</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>:
+       <literal>"registry.suse.de/suse/sle-15-sp3/update/bci/containerfile/suse/rmt-server"</literal>
```

Comment:
This is an internal url, it will not be reachable for anyone outside of the SUSE R&D network.
```suggestion
       <literal>"registry.suse.com/suse/rmt-server"</literal>
```